packages/dd-trace/src/encode/agentless-ci-visibility.js

```diff
@@ -198,6 +204,8 @@ class AgentlessCiVisibilityEncoder extends AgentEncoder {
     this._encodeMap(bytes, content.meta)
     this._encodeString(bytes, 'metrics')
     this._encodeMap(bytes, content.metrics)
+    this._encodeString(bytes, 'span_links')
+    this._encodeArray(bytes, content.links)
```

Comment:
I'm not sure these would be used by CI Visibility: we might drop them for now. Let me ask around

Reply:
yeah let's try to keep these out of the CI Visibility events: would it work to simply remove the changes in `packages/dd-trace/src/encode/agentless-ci-visibility.js`?